synfig-studio/src/gui/app.h

```diff
@@ -51,6 +51,7 @@
 #include <synfig/layers/layer_bitmap.h>
 #include <synfig/string.h>
 #include <synfig/time.h>
+#include <synfig/threadpool.h>
```

Comment:
Why did you add it?
Neither `app.h` nor `app.cpp` need it.

Reply:
to change the number of cores by calling a function 
https://github.com/SushantAA/synfig/blob/issue-1410/synfig-core/src/synfig/threadpool.cpp#L280

Reply:
I tried removing it but it showed a build error

Reply:
You should read the build error to check what file misses this header and fix it properly.